openquake/server/db/actions.py

```diff
@@ -480,8 +475,17 @@ def get_calcs(db, request_get_dict, user_name, user_acl_on=False, id=None):
     else:
         time_filter = 1
 
-    jobs = db('SELECT * FROM job WHERE ?A AND %s ORDER BY id DESC LIMIT %d'
-              % (time_filter, limit), filterdict)
+    # user_acl_on is true if settings.ACL_ON = True or when the user is a
+    # Django super user
+    if user_acl_on:
+        users_filter = ("user_name IN ('%s')" %
+                        "', '".join([str(item) for item in allowed_users]))
```

Comment:
Let's hope we do not have a user called Bobby table... https://imgs.xkcd.com/comics/exploits_of_a_mom.png

Reply:
This is something I don't like either... there's any facility to sanitize input?

Reply:
Yes there is, the `?X` syntax, like `db('select * from job where user_name IN (?X)', ['michele', 'openquake'])`